crs_perception/src/nodes/localize_to_part.cpp

```diff
@@ -26,7 +26,6 @@ namespace crs_perception
       // parameters
       mesh_num_samples_ = this->declare_parameter("mesh_num_samples", 100000);
       leaf_size_ = this->declare_parameter("leaf_size", 1.0);
-      camera_frame_ = this->declare_parameter("camera_frame", "/camera");
       part_frame_ = this->declare_parameter("part_frame", "/part");
```

Comment:
In ROS1 the "/" character meant the parameter existed in a global scope, is that still the case in ROS2.